Azure-Samples/chat-with-your-data-solution-accelerator
# refactor: make document loading pep8 compliant.

## Purpose

* Make document chunking pep8 compliant.
* Resolves #853 

## Does this introduce a breaking change?



## How to Test



* Run the tests:

```
poetry run pytest -m "not azure and not functional"    
poetry run pytest -m "functional"  
```

* Run the application locally and go to http://localhost:8502/Ingest_Data
* Upload a document on the above endpoint (Uploaded a JPG image)
* Verify that the file is present in the azure container on Blob Storage

![SCR-20240510-jtkn](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/4739534/293aee89-2bc5-40d8-947e-407fbda70333)


## What to Check
Verify that the following are valid
* Verify that the file is present in the azure container on Blob Storage
